bentoml/repository/__init__.py

```diff
@@ -85,40 +110,43 @@ def get(self, bento_name, bento_version=None):
             )
         return saved_path
 
-    def list(self, bento_name=None):
-        raise NotImplementedError
+    def dangerously_delete(self, bento_name, bento_version):
+        saved_path = os.path.join(self.base_path, bento_name, bento_version)
+        return shutil.rmtree(saved_path)
 
 
-class S3Repository(BaseRepository):
-    def __init__(self):
-        raise NotImplementedError
+class _S3BentoRepository(BentoRepositoryBase):
```

Comment:
suggestion: later on, we probably want to move both _s3 and _local repository implementations to different files